src/TMCStepper.h

```diff
@@ -370,6 +370,7 @@ class TMC2130Stepper : public TMCStepper {
 		static uint32_t spi_speed; // Default 2MHz
 		static uint32_t cs_low_time;
 		const uint16_t _pinCS;
+		SPIClass* TMC_HW_SPI;
```

Comment:
Can you pass this in as a reference instead of as a pointer?

It's better practice, to avoid any nullptr errors that could arise.